### PR TITLE
OHRI-2064: Add documentation for UI-select Extended

### PR DESCRIPTION
**Pull request by alaboso**
OHRI-2064: Add documentation for UI-select Extended with png illustration

### CODE REVIEW

**Review by otienodominic — COMMENTED** [review]
I've noticed this pull request and I'm interested in providing a review. May I have your permission to proceed with a detailed review? Looking forward to contributing to this project. Thank you! @ebambo @alaboso

File: pages/docs/field-types-reference.mdx [hunk 1]
```diff
@@ -514,3 +514,43 @@ Renders:
 
 ![Radio field](/camera-capture.png)
 
+## ui-select-extended
+
+The ui-select-extended is a searchable combo box used to load values from a data source. It works with both inbuilt datasources and custom datasources.
+
+The currently inbuilt datasources are:
+    - `location_datasource` - A `LocationDataSource` component that fetches `Location` entities with name matching search term entered in the combo box
+    - `drug_datasource` - A `ConceptDataSource` component that fetches `Concept` entities with name matching search term entered in the combo box
```
Comment (kajambiya) [inline pages/docs/field-types-reference.mdx:523]:
```suggestion
    - `drug_datasource` - A `ConceptDataSource` component that fetches `drug Concept` entities with name matching search term entered in the combo box
```

File: pages/docs/field-types-reference.mdx [hunk 2]
```diff
@@ -514,3 +514,43 @@ Renders:
...
+    - `problem_datasource` - A `ConceptDataSource` component that fetches `Concept` entities with name matching search term entered in the combo box
```
Comment (kajambiya) [inline pages/docs/field-types-reference.mdx:524]:
```suggestion
    - `problem_datasource` - A `ConceptDataSource` component that fetches `diagnosis and symptom Concept` entities with name matching search term entered in the combo box
```

**Review by kajambiya — COMMENTED** [review]
Hi @alaboso 
This is good. Just a few small suggestions. Also I would suggest we add 2 GIFs' one for preloaded and the other for searchable.

**Review by kajambiya — APPROVED** [review]

**Pull request merged**